src/__tests__/__snapshots__/wait-for-element-to-be-removed.js.snap

```diff
@@ -0,0 +1,51 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`it returns error immediately if there callback returns falsy value or error before any mutations 1`] = `
+Array [
+  [ReferenceError: timer is not defined],
+]
+`;
+
+exports[`it returns error immediately if there callback returns falsy value or error before any mutations 2`] = `
+Array [
+  [ReferenceError: timer is not defined],
```

Comment:
This error is actually being thrown because we call `onDone` before the `timer` is defined. Almost missed that because we were using snapshots and the tests are separated from the assertions...